views/cart.html

```diff
@@ -0,0 +1,279 @@
+                    <blockquote class="flex flex-wrap gap pb-1">
+                      <p class="font-bold font-sm ">Mehul Satardekar</p>
+                      <span class="col-gray">(mehulsatardekar@gmail.com)</span>
+                    </blockquote>
+                    <figcaption class="blockquote-footer font-mid-light">
+                      <cite title="Source Title col-gray">You are securely logged in</cite>
+                    </figcaption>
+                  </figure>
+            </div>
+
+            <div class=" card-py gap shadow-sm mb-1 flex flex-column  pb-3">
+                
+                <div class="flex flex-between flex-wrap flex-align-item-center">
+                    <h1 class="font-mid-bold">Delivery Address</h1>
+                    <button class="btn-outline-primary btn btn-sm">Change</button>
+
+                </div>
+                <div class="flex flex-column gap-1">
+                    <span class="font-md-light">Deliver to:  <span class="label-text">John Doe, 560078</span></span>
+                    <span class="label-text">#1/4 , 100ft Ring Road, Jp Nagar - 4 Phase, Dollars Colony, Bangalore</span>
+
+                </div>
+
+            </div>
+
+
+
+
+           <div class="ui-card-default py">
+              <p class="pb-1 flex flex-align-item-center gap-1"> 
+                  <span class="font-bold font-sm ">Order Summary</span>
+                 <span class="col-gray">(3 dishes.)</span>
+            </p>
+            <div class="ui-card-list card-py gap shadow-sm mb-1">
+                
+                <div class="ui-card-list-img relative">
+                    <img src="https://res.cloudinary.com/swiggy/image/upload/fl_lossy,f_auto,q_auto,w_508,h_320,c_fill/lite97zgd44crsd6znxo"
+                        alt="chinese work" loading="lazy">
+                    <div class="card-badge">
+                        <span>Pro- Get 20% off</span>
+                    </div>
+                </div>
+                <div class="ui-card-list-content">
+                    <span class="font-bold ">Chinese Wok (Wok Express) </span>
+                    <div>
+                        <span>&#x20b9;</span>
+                        <span class="font-md-light">254</span>
+                    </div>
+                    <span class="font-md-light">Light eater - choice of any 1 rice/noodles, 1 sauce, 4
+                        vegetables and 2 seasoning.</span>
+                </div>
+                <div class="ui-card-list-button gap flex flex-column">
+                    <div class="flex flex-row gap">
+                        <button class="btn-outline-primary btn pl-04 pr-04 pt-03">
+                            <span class="material-icons icon-menu">
+                                add
+                            </span>
+                        </button>
+    
+                        <input type="number" class="input input-qty" value="2" min="1">
```

Comment:
Change- As you are using the type="number" the spinner provides click up and down arrows to select a number. So either you can remove that or remove +and - buttons.

Reply:
done

Reply:
@vanshitamoolchandani also Thanks for the reference link😁